doc/nrf/releases/release-notes-1.4.0.rst

```diff
@@ -0,0 +1,1019 @@
+
+* Updated:
+
+  * Changed :ref:`bt_mesh_light_ctrl_readme`, so that the light control regulator now uses floating point.
+  * Removed the necessity of setting several Kconfig options when using nRF5340 PDK with :ref:`bluetooth_mesh_light` or :ref:`bluetooth_mesh_light_switch`.
+
+* Fixed several bugs and improved documentation.
+
+nRF IEEE 802.15.4 radio driver
+------------------------------
+
+* Added support for multiprotocol with :ref:`nrfxlib:softdevice_controller`.
+
+Thread
+------
+
+* Added:
+
+  * Production support for nRF52833 and nRF52840 DKs.
+  * Dynamic multiprotocol support with Bluetooth LE.
+  * Support for :ref:`thread_architectures_designs_cp_ncp` architecture.
+  * Support for :ref:`ug_thread_vendor_hooks`.
+  * :ref:`ot_ncp_sample` sample - This sample demonstrates the usage of OpenThread Network Co-Processor architecture inside the Zephyr environment.
+    It has the following characteristics:
+
+    * Extendable with vendor hooks
+    * Hardware cryptography acceleration support
+    * Support for :ref:`Spinel logging <ug_logging_backends>` as a default logger backend
+    * UART hardware flow control (HWFC) enabled by default
+
+  * Initial support for :ref:`Thread 1.2 functionalities <thread_ug_supported_features_v12>` related to SED implementation and reduction of power consumption.
+
+* Enabled Thread v1.1 certification by inheritance.
+  For more information about certification, see :ref:`ug_thread_cert`.
+* Updated:
+
+  * :ref:`ot_cli_sample` sample:
+
+    * Switched to RNG peripheral as an entropy source.
+    * Added support for :ref:`Thread Certification Test harness <ug_thread_cert>`.
+    * Added hardware cryptography acceleration support.
+    * Enabled UART HWFC by default.
+    * Added the overlay for :ref:`Thread v1.2 support <ot_cli_sample_thread_v12>`.
+
+  * :ref:`coap_client_sample` sample:
+
+    * Switched to RNG peripheral as an entropy source.
+    * Optimized power in SED mode.
+    * Added hardware cryptography acceleration support.
+    * Added :ref:`multiprotocol support <coap_client_sample_multi_ext>`.
+    * Enabled UART HWFC by default.
+
+  * :ref:`coap_server_sample` sample:
+
+    * Switched to RNG peripheral as an entropy source.
+    * Added hardware cryptography acceleration support.
+    * Enabled UART HWFC by default.
+
+Zigbee
+------
+
+* Added:
+
+  * Production support for nRF52833 and nRF52840.
+  * :ref:`lib_zigbee_fota` (DFU) support for nRF52840.
+  * Support for Command Line Interface (:ref:`lib_zigbee_shell`).
+    This is a port from the nRF5 SDK for Thread and Zigbee.
+
+* Updated:
+
+  * :ref:`zigbee_light_switch_sample` sample with :ref:`multiprotocol extension <zigbee_light_switch_sample_nus>` based on Bluetooth LE :ref:`nus_service_readme`.
+  * Updated :ref:`zboss` to version ``v3_3_0_4+10_06_2020``.
```

Comment:
If this PR gets merged: https://github.com/nrfconnect/sdk-nrf/pull/3205
Please update the version string to `v3_3_0_5+10_06_2020`